airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/AbstractJdbcDestination.java

```diff
@@ -84,11 +87,39 @@ public AirbyteConnectionStatus check(final JsonNode config) {
     }
   }
 
+  /**
+   * This method is deprecated. It verifies table creation, but not insert right to a newly created
+   * table. Use attemptSQLCreateTableThenInsertDummyRecThenDropThisTableOperations with
+   * isNeedTryMakeTestDataInsertInTable argument instead.
+   */
+  @Deprecated
   public static void attemptSQLCreateAndDropTableOperations(final String outputSchema,
                                                             final JdbcDatabase database,
                                                             final NamingConventionTransformer namingResolver,
                                                             final SqlOperations sqlOps)
       throws Exception {
+    attemptSQLCreateTableThenInsertDummyRecThenDropThisTableOperations(outputSchema, database, namingResolver, sqlOps, false);
+  }
+
+  /**
+   * Verifies if provided creds has enough permissions. Steps are: 1. Create schema if not exists. 2.
+   * Create test table. 3. Insert dummy record to newly created table if
+   * "isNeedTryToInsertDummyRecordInNewlyCreatedTable" set to true. 4. Delete table created on step 2.
+   *
+   * @param outputSchema - schema to tests against.
+   * @param database - database to tests against.
+   * @param namingResolver - naming resolver.
+   * @param sqlOps - SqlOperations object
+   * @param isNeedTryToInsertDummyRecordInNewlyCreatedTable - set true if need to make attempt to
```

Comment:
This variable is still a little confusing, maybe a better one could be something like:

* `shouldInsertDummyRecord`
* `isInsertOperationSupported`

Not sure how you want to phrase this variable but overall it's still not the easiest to read right now

Reply:
Updated to "attemptInsert" as Greg asked in the comment above. Thank